server/resourcemanager/resource_manager.go

```diff
@@ -110,18 +110,21 @@ func (m *manager[T]) RegisterRoutes(r *mux.Router) *mux.Router {
 	enabledOps := m.EnabledOperations()
 
 	if slices.Contains(enabledOps, OperationList) {
-		subrouter.HandleFunc("/", m.list).Methods(http.MethodGet).Name("list")
+		subrouter.HandleFunc("", m.list).Methods(http.MethodGet).Name("list")
 	}
+
 	if slices.Contains(enabledOps, OperationCreate) {
-		subrouter.HandleFunc("/", m.create).Methods(http.MethodPost).Name(fmt.Sprintf("%s.Create", m.resourceType))
```

Comment:
The obscured hidden bug haha great catch!

Reply:
As @danielbdias likes to say: This like of code owns me 3 hours of debugging